docs/site/translation/date-time/date-time-patterns.md

```diff
@@ -286,47 +286,53 @@ What you want to pay attention to are:
 - Remove a space if your language does not use spaces, which is common for many East Asian languages
 - Add a comma, or other punctuation that your language requires between the patterns
 
-There are four formats: full, long, medium, and short. _Each of these may come in two variants_:
+There are four format length styles: full, long, medium, and short. _Each of these may come in up to three variants_:
 
-- _The “-atTime” variant, which specifies a date at a particular time, typically for an event. In the longer formats (such as full and long), this “-atTime” form may have a combining word betwen the date and the time, for example “{1} 'at' {0}” to produce an English example like “Sunday, September 25 at 1:30 PM”)._
 - _The standard variant, which is used for multuple purposes and typically does not include any literal text, for example “{1}, {0}”. Usage examples include:_
 	- _Wall clock time: “Sunday, September 25, 1:30 PM”_
 	- _Combining a date with a time range: “Sunday, September 25, 1:30 – 3:00 PM”_
+- _The “-atTime” variant, which specifies a fixed date at a particular time, typically for an event. In the longer formats (such as full and long),
+this “-atTime” form may have a combining word betwen the date and the time, for example “{1} 'at' {0}” to produce an English example like “Sunday, September 25 at 1:30 PM”).
+If there is no “-atTime” variant, the standard variant will be used for this combination._
+- _The “-relative” variant (new in CLDR 48), which specifies a relative date at a particular time, typically for an event. In the longer formats (such as full and long),
+this “-atTime” form may have a combining word betwen the date and the time, for example “{1} 'at' {0}” to produce an English example like “tomorrow at 1:30 PM”).
```

Comment:
Should this say "this '-relative' form"?

Reply:
Yes, thanks, good catch.